fury/ui/core.py

```diff
@@ -325,9 +327,13 @@ def mouse_move_callback(i_ren, obj, self):
             self.on_middle_mouse_button_dragged(i_ren, obj, self)
 
     @staticmethod
-    def key_press_callback(i_ren, obj, self):
+    def on_key_press_callback(i_ren, obj, self):
```

Comment:
if you change that, please, check the whole codebase. I suppose, your PR is failing because of that. all `UI` depends on this